src/framework/lib/genesisframework/utils.rb

```diff
@@ -44,9 +44,12 @@ def self.facter
         end
       end
 
-      def self.log subsystem, message
-        logline = subsystem.to_s + " :: " + message
-        puts logline
+      def self.log subsystem, message, level = nil
```

Comment:
I don't really see how to do it right now ...

Should we use and enforce the `collins_client` api logging levels and pass them to the external logging modules or have you something else in mind ?